doc/changes/names.inc

```diff
@@ -166,6 +166,8 @@
 
 .. _Guillaume Favelier: https://github.com/GuillaumeFavelier
 
+.. _Hakimeh Aslsardroud: https://www.researchgate.net/profile/Hakimeh-Pourakbari
```

Comment:
Please double-check this link @moritz-gerster , it's more of a guess from my end

Reply:
Looks good @larsoner! Is there anything else I should do?